articles/cognitive-services/Face/QuickStarts/Python.md

```diff
@@ -4,317 +4,141 @@ description: Get information and code samples to help you quickly get started us
+    plt.text(origin[0], origin[1], "%s, %d"%(fa["gender"].capitalize(), fa["age"]), fontsize=20, weight="bold", va="bottom")
+_ = plt.axis("off")
 ```
-#### Face Detect response
-
-A successful response is returned in JSON. Following is an example of a successful response: 
-
-```json
-Response:
-[
-  {
-    "faceAttributes": {
-      "accessories": [],
-      "age": 22.9,
-      "blur": {
-        "blurLevel": "low",
-        "value": 0.06
-      },
-      "emotion": {
-        "anger": 0.0,
-        "contempt": 0.0,
-        "disgust": 0.0,
-        "fear": 0.0,
-        "happiness": 0.0,
-        "neutral": 0.986,
-        "sadness": 0.009,
-        "surprise": 0.005
-      },
-      "exposure": {
-        "exposureLevel": "goodExposure",
-        "value": 0.67
-      },
-      "facialHair": {
-        "beard": 0.0,
-        "moustache": 0.0,
-        "sideburns": 0.0
-      },
-      "gender": "female",
-      "glasses": "NoGlasses",
-      "hair": {
-        "bald": 0.0,
-        "hairColor": [
-          {
-            "color": "brown",
-            "confidence": 1.0
-          },
-          {
-            "color": "black",
-            "confidence": 0.87
-          },
-          {
-            "color": "other",
-            "confidence": 0.51
-          },
-          {
-            "color": "blond",
-            "confidence": 0.08
-          },
-          {
-            "color": "red",
-            "confidence": 0.08
-          },
-          {
-            "color": "gray",
-            "confidence": 0.02
-          }
-        ],
-        "invisible": false
-      },
-      "headPose": {
-        "pitch": 0.0,
-        "roll": 0.1,
-        "yaw": -32.9
-      },
-      "makeup": {
-        "eyeMakeup": true,
-        "lipMakeup": true
-      },
-      "noise": {
-        "noiseLevel": "low",
-        "value": 0.0
-      },
-      "occlusion": {
-        "eyeOccluded": false,
-        "foreheadOccluded": false,
-        "mouthOccluded": false
-      },
-      "smile": 0.0
-    },
-    "faceId": "49d55c17-e018-4a42-ba7b-8cbbdfae7c6f",
-    "faceRectangle": {
-      "height": 162,
-      "left": 177,
-      "top": 131,
-      "width": 162
-    }
-  }
-]
-```
-## Create a Person Group with Face API using Python <a name="Create"> </a>
-Use the [Person Group - Create a Person Group method](https://westcentralus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395244) 
-to create a person group with specified personGroupId, name, and user-provided userData. A person group is one of the most important parameters for the Face - Identify API. The Identify API searches for persons' faces in a specified person group. 
 
-#### Person Group - create a Person Group example
+For more information on the Microsoft Cognitive Face APIs, see [here](https://westus.dev.cognitive.microsoft.com/docs/services/563879b61984550e40cbbe8d/operations/563879b61984550f30395236) for the full documentation of the REST APIs.
```

Comment:
s/full documentation/full API reference/g

Reply:
I just realized the very same URL is provided a few sentences above so I got rid of it.